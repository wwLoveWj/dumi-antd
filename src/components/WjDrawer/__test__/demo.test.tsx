import demoTest from '../../../tests/shared/demoTest';

demoTest('MsDrawer');
