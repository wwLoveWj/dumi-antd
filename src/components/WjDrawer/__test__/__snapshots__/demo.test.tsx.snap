// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MsDrawer 组件 demo 测试 正确渲染 MsDrawer basic.tsx 1`] = `
<div>
  <button
    class="ant-btn ant-btn-default"
    type="button"
  >
    <span>
      打 开
    </span>
  </button>
</div>
`;

exports[`MsDrawer 组件 demo 测试 正确渲染 MsDrawer loading.tsx 1`] = `
<div>
  <button
    class="ant-btn ant-btn-default"
    type="button"
  >
    <span>
      打 开
    </span>
  </button>
</div>
`;

exports[`MsDrawer 组件 demo 测试 正确渲染 MsDrawer promise.tsx 1`] = `
<div>
  <div
    class="ant-space ant-space-horizontal ant-space-align-center"
  >
    <div
      class="ant-space-item"
      style="margin-right: 8px;"
    >
      <button
        class="ant-btn ant-btn-default"
        type="button"
      >
        <span>
          Promise成功
        </span>
      </button>
    </div>
    <div
      class="ant-space-item"
    >
      <button
        class="ant-btn ant-btn-default"
        type="button"
      >
        <span>
          Promise失败
        </span>
      </button>
    </div>
  </div>
</div>
`;

exports[`MsDrawer 组件 demo 测试 正确渲染 MsDrawer size.tsx 1`] = `
<div>
  <div
    class="ant-space ant-space-horizontal ant-space-align-center"
  >
    <div
      class="ant-space-item"
      style="margin-right: 8px;"
    >
      <button
        class="ant-btn ant-btn-default"
        type="button"
      >
        <span>
          small(500px)
        </span>
      </button>
    </div>
    <div
      class="ant-space-item"
      style="margin-right: 8px;"
    >
      <button
        class="ant-btn ant-btn-default"
        type="button"
      >
        <span>
          middle(700px)
        </span>
      </button>
    </div>
    <div
      class="ant-space-item"
      style="margin-right: 8px;"
    >
      <button
        class="ant-btn ant-btn-default"
        type="button"
      >
        <span>
          large(900px)
        </span>
      </button>
    </div>
    <div
      class="ant-space-item"
    >
      <button
        class="ant-btn ant-btn-default"
        type="button"
      >
        <span>
          width=999px
        </span>
      </button>
    </div>
  </div>
</div>
`;
